spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/RestAPIBuilder.java

```diff
@@ -62,7 +62,7 @@ public String buildKVApi(@Nullable String prefix, @Nullable String label) {
             this.addParam("key", prefix);
         }
 
-        label = label == null ? "%00" : label; // label=%00 matches null label
+        label = StringUtils.isEmpty(label) ? "%00" : label; // label=%00 matches null label
```

Comment:
In another PR, you used `hasText`. Any specific reason to use `isEmpty` here

Reply:
Should be some typo... updated.